scicat_schemas/dataset.schema.json

```
@@ -0,0 +1,47 @@
+{
+  "pid": "20.500.12269/e3690b21-ee8c-40d6-9409-6b6fdca776d2",
+  "datasetName": "this is a dataset",
+  "description": "this is the description of the dataset",
```

How about we turn this into [jinja](https://jinja.palletsprojects.com/en/3.1.x/api/#basics) template?

```suggestion
  "datasetName": "{{ dataset_name }}",
  "description": "{{ dataset_description }}",
```